allure-generator/src/main/javascript/components/modal/ModalView.hbs

```diff
@@ -2,9 +2,7 @@
     <div class="modal__window">
         <h2 class="{{b cls 'title'}}">
             <span>{{title}}</span>
-            <div class="pane__controls">
-                <span class="modal__close fa fa-close" data-tooltip="Close"></span>
-            </div>
+            <button class="modal__close fa fa-close" data-tooltip="Close"></button>
```

Comment:
what is the point of the change? I personally dont like the button borders in here

Reply:
is it ok to bring the X on the same line as the attachment name atleast? or do you prefer having the pane__controls div underneath, i guess if we add some new buttons later it may be best?

Reply:
> or do you prefer having the pane__controls div underneath

nope, thats a bug. X should be on the same line, float to the right